sql/moz-fx-data-shared-prod/org_mozilla_mozregression_derived/mozregression_aggregates_wchan1_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: mozregression aggregates wchan1
+description: |-
+  Aggregated metrics of mozregression usage
+owners:
+- wichan@mozilla.com
+labels:
+  incremental: true
+  public_json: true
+  public_bigquery: true
+  review_bugs:
+  - '1691105'
```

Comment:
I linked to the same review bug in the instructions since it is the same dataset.  Let me know if I need to create a new bug instead.